src/ScottPlot4/ScottPlot/Plottable/Image.cs

```diff
@@ -14,18 +14,23 @@ public class Image : IPlottable
         public bool IsVisible { get; set; } = true;
 
         /// <summary>
-        /// Position of the primary corner (based on Alginment)
+        /// Position of the primary corner (based on Alignment)
         /// </summary>
         public double X { get; set; }
 
         /// <summary>
-        /// Position of the primary corner (based on Alginment)
+        /// Position of the primary corner (based on Alignment)
         /// </summary>
         public double Y { get; set; }
 
+        /// <summary>
+        /// Size of the image, in axis units. If null, the image will use pixel units.
+        /// </summary>
+        public SizeF? SizeInAxisUnits { get; set; } = null;
```

Comment:
Nice, I hadn't remembered that System.Drawing had this struct when I made my comment.

Although this may be a downside when we transition off of System.Drawing (see #1036 if you're interested). There are already a lot of places where we expose System.Drawing structs directly to the user, so unless @swharden disagrees, I would leave this using `SizeF` for simplicity's sake. Just thought I would note it.

Reply:
There is still a problem, the user can't use img.SizeInAxisUnits.Width to set the width, they must set the whole img.SizeInAxisUnits. I don't like that, and that's why I converted this PR to draft. I'm thinking about adding a class for size, but it seems to be too heavy.

Reply:
Ah, that is annoying. And more importantly I think that it would be confusing to most users, who wouldn't know nor care that `SizeF` is a struct. Perhaps a class is the way to go.

Reply:
> using System.Drawing structs ... may be a downside when we transition off of System.Drawing

I'm fine with this for now - it's littered throughout the existing code base and it will be straightforward for me to make my replacements all at once if/when these plottables are translated to a new rendering system 👍